contentpacks/__main__.py

```diff
@@ -31,16 +32,24 @@ def make_language_pack(lang, version, sublangargs, filename):
     html_exercise_path, translated_html_exercise_ids = retrieve_html_exercises(html_exercise_ids, lang)
 
     # now include only the assessment item resources that we need
-    # all_assessment_resources = get_full_assessment_resource_list()
-    # included_assessment_resources = filter_unneeded_assessment_resources(all_assessment_resources, exercise_data)
+    all_assessment_data, all_assessment_files = retrieve_all_assessment_item_data()
 
-    # node_data = remove_untranslated_exercises(exercise_data, translated_html_exercise_ids, assessment_data)
+    assessment_data = translate_assessment_item_text(all_assessment_data, content_catalog)
 
-    # node_data = remove_unavailable_topics(node_data)
+    node_data = remove_untranslated_exercises(node_data, translated_html_exercise_ids, assessment_data)
+
+    node_data = remove_unavailable_topics(node_data)
+
+    included_assessment_data = filter_unneeded_assessment_resources(assessment_data, node_data)
 
     pack_metadata = generate_kalite_language_pack_metadata(lang, version, interface_catalog, content_catalog)
 
-    bundle_language_pack(str(filename), node_data, interface_catalog, interface_catalog, pack_metadata)
+    if lang != "en":
```

Comment:
Instead of a hardcoded check for "en", can you instead have assessment items bundled in by default, and then have it empty when `--no-assessment-items` is specified?

So basically, change the module docstring above by adding `--no-assessment-items`:

```bash
makecontentpacks ka-lite <lang> <version> [--subtitlelang=subtitle-lang --contentlang=content-lang --interfacelang=interface-lang --videolang=video-lang --out=outdir --no-assessment-items]
```

And then there should be a new key in `args` [here](https://github.com/fle-internal/content-pack-maker/pull/6/files#diff-6663f24947cd67b72824e55d38374fc6L60), called `args['--no-assessment-items']`, which you can now propagate to this line of code.